RUFAS/general_constants.py

```diff
@@ -96,6 +96,67 @@ class GeneralConstants:
     HECTARES_PER_SQUARE_METER = 10_000
     SQUARE_METERS_TO_HECTARES = 1 / HECTARES_PER_SQUARE_METER
 
+    EFFECTIVENESS_OF_MICROBIAL_DECOMPOSITION_RATE: float = 2.37e-3
+    """The rate of effectiveness of microbial decomposition."""
+
+    FIRST_ORDER_DECAYING_COEFFICIENT: float = 0.1
+    """The first order decaying coefficient."""
+
+    DECOMPOSITION_TEMPERATURE: float = 60.0
+    """The temperature of the inner decomposing material layer at which microbial growth and decomposition is
+    maximized (C)."""
+
+    DEFAULT_MOLE_FRACTION_OF_OXYGEN: float = 0.15
+    """The default mole fraction of oxygen in the air within the decomposing material layer."""
+
+    OXYGEN_HALF_SATURATION_CONSTANT: float = 0.02
+    """The half saturation constant of Oxygen gas (O2)"""
+
+    AMBIENT_AIR_MOLE_FRACTION_OF_OXYGEN: float = 0.21
+    """The mole fraction of oxygen in ambient air."""
+
+    METHANE_FACTOR: float = 0.67
+    """Unit conversion factor for methane from :math:`m^3` to kg (unitless)."""
+
+    MCF_CONSTANT_A: float = 0.0625
+    """
+    Parameter estimate (unitless) of a regression using IPCC data (2006) used in the
+    Methane Conversion Factor (MCF) calculation. The coefficient scales the ambient barn temperature.
+    """
+
+    MCF_CONSTANT_B: float = 0.25
+    """
+    Parameter estimate (unitless) of a regression using IPCC data (2006) used in the
+    Methane Conversion Factor (MCF) calculation. The coefficient is a constant offset.
+    """
+
+    DEFAULT_PH_FOR_AMMONIA: float = 7.5
+    """Default pH for ammonia (unitless)."""
```

Comment:
```suggestion
    DEFAULT_STORED_MANURE_PH: float = 7.5
    """Default pH of manure in slurry storage or anaerobic lagoon (unitless)."""
```
Sorry for the additional name change here. I have not added suggestions to change instances of this name, if you could please do a quick find and replace!